rbac/management/group/serializer.py

```diff
@@ -43,6 +43,21 @@ def get_roleCount(self, obj):
         """Role count for the serializer."""
         return obj.role_count()
 
+    def is_custom_default_group(self, group):
+        """Check if the group is custom default or not."""
+        if group.platform_default and not group.system:
+            return True
+        return False
+
+    def restrict_custom_default_group_renaming(self, group):
+        """Restrict users from changing the name or description of the Custom default group."""
+        is_custom_default = self.is_custom_default_group(group)
+        if is_custom_default:
+            key = "detail"
+            message = "Updating the name or description of Custom default group is restricted"
```

Comment:
nitpick: it would be nice to have the name of the group in single quotes to distinguish what is name group